README.md

```diff
@@ -2,11 +2,26 @@
 
 wf-recorder is a utility program for screen recording of `wlroots`-based compositors (more specifically, those that support `wlr-screencopy-v1` and `xdg-output`). Its dependences are `ffmpeg`, `wayland-client` and `wayland-protocols`.
 
-To compile:
+# installation
+
+## archlinux
+
+Arch users can use [wf-recorder-git](https://aur.archlinux.org/packages/wf-recorder-git/) from the AUR.
+```
+yay -S wf-recorder-git #replace yay by your favorite AUR helper
+```
+
+
+## from source
 
 ```
 git clone https://github.com/ammen99/wf-recorder && cd wf-recorder
 meson build --prefix=/usr --buildtype=release
 ninja -C build
 ```
-Now you can just run `./build/wf-recorder` or install it with `sudo ninja -C build install`. Use `-f <filename>` to specify the output file. In case of multiple outputs, you'll first be prompted to select the output you want to record. If you know the output name beforehand, you can use the `-o <output name>` option. 
+Now you can just run `./build/wf-recorder` or install it with `sudo ninja -C build install`.
+
+# usage
+Use `-f <filename>` to specify the output file. In case of multiple outputs, you'll first be prompted to select the output you want to record. If you know the output name beforehand, you can use the `-o <output name>` option. 
+
+You can use `wf-recorder -g (slurp)` to select and limit the recording to a part of the screen.
```

Comment:
Should be `wf-recorder -g $(slurp)`

Reply:
actually I think it should be `wf-recorder -g "$(slurp)"` at least that's what I use for zsh

Reply:
also I think you should note that this requires `https://github.com/emersion/slurp`

Reply:
You're right, I use fish and it happens to me all the time. My bad.